notebooks/contributions_quantification.ipynb

```diff
@@ -6,7 +6,7 @@
       "name": "contributions_quantification.ipynb",
```

Comment:
<p>Why doesn't this data match the printout above? </p><p> </p><blockquote>Michelle Vukovich       253939 </blockquote><blockquote>Tim Cole                 21412 </blockquote><blockquote>Beth Josephson           10948 </blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/iobis/pyobis/pull/67/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
I don't know why they aren't matching. I queried through dataset endpoint and it resulted into those printed values, but with the same parameters in the search query when I fetched occurrence records, the number was different. The total length of the occurrence records fetched was 104621 which is way smaller than it should have been. When I run a similar visualization on the data from dataset endpoint I get a graph like: 

![image](https://user-images.githubusercontent.com/36472216/186161233-587dfe8a-fee4-4528-860f-eb838c0840ba.png)